nms/app/packages/magmalte/app/components/lte/LteMetrics.js

```diff
@@ -262,37 +247,38 @@ export default function () {
     tabNames.push('grafana');
   }
 
-  const currentTab = findIndex(tabNames, route =>
-    location.pathname.startsWith(match.url + '/' + route),
-  );
+  let tabList = [];
+  if (!grafanaEnabled) {
+    tabList = [
+      {
+        component: {NestedRouteLink},
+        label: 'Gateways',
+        to: '/gateways',
+      },
+      {
+        component: {NestedRouteLink},
+        label: 'Internal',
+        to: '/internal',
+      },
+    ];
+  } else {
+    tabList = [
+      {
+        component: {NestedRouteLink},
+        label: 'Grafana',
+        to: '/grafana',
+      },
+      {
+        component: {NestedRouteLink},
+        label: 'Explorer',
+        to: '/explorer',
+      },
+    ];
```

Comment:
Could we swap the order of these? The `Explorer` page is more explanatory and loads faster, so could be more convenient for a relatively new user.

Reply:
Sure. I can do that.